examples/gallery/simple_differential_rotation.py

```diff
@@ -0,0 +1,49 @@
+"""
+============================
+Simple Differential Rotation
+============================
+
+This example shows the effect of differential rotation.
```

Comment:
This could be expanded a little bit to describe what differential rotation is and how this demonstrates it.